src/test/ui/async-await/in-trait/async-lifetimes.rs

```diff
@@ -0,0 +1,18 @@
+// edition: 2021
+
+#![feature(async_fn_in_trait)]
+#![allow(incomplete_features)]
+
+trait MyTrait<'a, 'b, T> {
+    async fn foo(&'a self, key: &'b T) -> (&'a Self, &'b T);
```

Comment:
This is basically identical to src/test/ui/async-await/in-trait/async-lifetimes-and-bounds.rs, so add the same known-bug.